Assets/Python/UniquePowers.py

```diff
@@ -233,17 +206,17 @@ def tamilConquestUP(self, iEnemy, iNumTargets=1):
 
 #------------------ARABIAN U.P.-------------------
 
-	def mughalUP(self, city):
+	def mughalUP(self, city): # Unused
 		return
-		pMughals = gc.getPlayer(iMughals)
+		# pMughals = gc.getPlayer(iMughals)

```

Comment:
pMughals is used right in the next line, that's is going to fail (if the method is ever used again).